src/lib_c/x86_64-windows-msvc/c/winreg.cr

```diff
@@ -0,0 +1,114 @@
+lib LibC
+  enum REGSAM
+    # Combines the `STANDARD_RIGHTS_REQUIRED`, `QUERY_VALUE`, `SET_VALUE`, `CREATE_SUB_KEY`, `ENUMERATE_SUB_KEYS`, `NOTIFY`, and `CREATE_LINK` access rights.
+    ALL_ACCESS = 0xf003f
+
+    # Reserved for system use.
+    CREATE_LINK = 0x00020
+
+    # Required to create a subkey of a registry key.
+    CREATE_SUB_KEY = 0x00004
+
+    # Required to enumerate the subkeys of a registry key.
+    ENUMERATE_SUB_KEYS = 0x00008
+
+    # Equivalent to `READ`.
+    EXECUTE = 0x20019
+
+    # Required to request change notifications for a registry key or for subkeys of a registry key.
+    NOTIFY = 0x00010
+
+    # Required to query the values of a registry key.
+    QUERY_VALUE = 0x00001
+
+    # Combines the `STANDARD_RIGHTS_READ`, `QUERY_VALUE`, `ENUMERATE_SUB_KEYS`, and `NOTIFY` values.
+    READ = 0x20019
+
+    # Required to create, delete, or set a registry value.
+    SET_VALUE = 0x00002
+
+    # Indicates that an application on 64-bit Windows should operate on the 32-bit registry view. This flag is ignored by 32-bit Windows.
+    # This flag must be combined using the OR operator with the other flags in this table that either query or access registry values.
+    # Windows 2000: This flag is not supported.
+    WOW64_32KEY = 0x00200
+
+    # Indicates that an application on 64-bit Windows should operate on the 64-bit registry view. This flag is ignored by 32-bit Windows.
+    # This flag must be combined using the OR operator with the other flags in this table that either query or access registry values.
+    # Windows 2000: This flag is not supported.
+    WOW64_64KEY = 0x00100
+
+    # Combines the STANDARD_RIGHTS_WRITE, `KEY_SET_VALUE`, and `KEY_CREATE_SUB_KEY` access rights.
+    WRITE = 0x20006
+  end
+
+  enum ValueType
+    NONE                       = 0
+    SZ                         = 1
+    EXPAND_SZ                  = 2
+    BINARY                     = 3
+    DWORD                      = 4
+    DWORD_LITTLE_ENDIAN        = DWORD
+    DWORD_BIG_ENDIAN           =  5
```

Comment:
minor, but weird alignment here (though it's probably `crystal tool format`)... the `4` is aligned different from the `5`, likely because the DWORD breaks up the 2 sequences.